internal/step/spinner.go

```diff
@@ -0,0 +1,58 @@
+package step
+
+import (
+	"fmt"
+	"github.com/briandowns/spinner"
+	"time"
+)
+
+func NewStepWithSpinner(msg string) Step {
+	s := spinner.New([]string{"/", "-", "\\", "|"}, time.Millisecond * 200)
+	s.Prefix = msg+" "
+	return  &stepWithSpinner{s, msg}
+}
+
+type stepWithSpinner struct {
```

Comment:
That is a nice new spinner, thanks for adding it. However, you have not replaced the usage of the old one, please apply it consistently to all. I can support you with that if you prefer.
I realized that the new spinner has a much more light-grey appearance on my terminal then the old one. Can you check if the tile/frame itself can be a bit more black..

Reply:
The spinner had foreground set to white. I've cleared all styles on spinner, now it's back to normal.

Reply:
looks fine now, thanks